src/taco-livereload/lib/utils/ATSRemover.js

```diff
@@ -17,7 +17,8 @@ module.exports = function(projectRoot, platform) {
     var self = this;
     this.Remove = function() {
         var configFolder = path.join(projectRoot, 'platforms', platform, multiPlatforms.getConfigFolder(platform));
-        glob.sync('**/*Info.plist', {
+        // We only want to modify a single plist file, not all of them.
```

Comment:
While this comment may make sense in the context of this PR/bug fix, it will only confuse anyone else reading the code who doesn't have that context. Please, remove it.